pages/about.html

```diff
@@ -4,9 +4,100 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="styles/style.css" rel=Stylesheet>
 </head>
-<body>
+<body> 
+  <header class="about-header">
+    <nav class="ss-bg">
+      <ul id="menu" class="about-menu">
+        <li><a href="http://127.0.0.1:5502/pages/index.html"><h3>Home</h3></a></li>
+        <li><a href="http://127.0.0.1:5502/pages/about.html"><h3>About</h3></a></li>
+        <li><img id="About-me-logo" alt="logo" src="../assets/ABOUT+ME+LOGO.png"></li>
+        <li><a href="http://127.0.0.1:5502/pages/portfolio.html"><h3>Portfolio</h3></a></li>
+        <li><a href="https://www.linkedin.com/in/wanjun-lan/"><h3>Contact</h3></a></li>
+      </ul>
+    </nav>
+  </header> 
+  <div id="div-grid">
+    <article id="article-grid">
+      <div class="article-grid-items">
+        <h2>Hi. I'm Cool. See Below.</h2>
+      </div>
+      <div class="article-grid-items">
+        <p><em>Software Developer Student, Biotech Researcher, Audiophile, Photographer, Dreamer</em></p>
+      </div> 
+    </article> 
+    <main id="main-grid">
+      <div class="main-wrapper">
+        <p>
+          Wanjun Lan is lover of many things including, but not limited to: spicy Thai, dry red wines, 
+          Imperial stouts, IPAs, whiskey tastings, outdoor adventures, backpacking the wilderness, 
+          international travel, Films: All Wes Anderson Films, Eternal Sunshine, Stranger Than Fiction, 
+          Wristcutters: A Love Story, Books: American Gods, The Wind Up Bird Chronicles, The Diving Bell 
+          and the Butterfly, The Fault In Our Stars, The Motorcycle Diaries, iced Americanos, good music, 
+          better conversations.
+        </p>
+      </div>
+      <div class="about-main-container">
+        <div>
+          <img class="me-photo" alt="fuji" src="../assets/Fuji.JPG">
```

Comment:
It would be good for alt text to be more descriptive. Alt text is used by screen readers for people who are blind to explain to them what the picture is. Just saying "Fuji" is not enough to help someone who is blind understand what is in the picture.